src/main/java/ch/njol/skript/structures/StructCommand.java

```diff
@@ -260,6 +263,23 @@ public boolean load() {
 
 		String description = entryContainer.get("description", String.class, true);
 
+		String prefix = entryContainer.getOptional("prefix", String.class, false);
```

Comment:
Set this to use the default value which is `skript` the original value, so when they don't specify the prefix it will be `skript`

Reply:
same as above